package_control/show_error.py

```diff
@@ -21,5 +34,31 @@ def show_error(string, params=None, strip=True, indent=None):
         If all lines should be indented by a set indent after being dedented
     """
 
+    global is_error_recentely_displayed
     string = text.format(string, params, strip=strip, indent=indent)
-    sublime.error_message(u'Package Control\n\n%s' % string)
+
+    if sublime:
+
+        if is_error_recentely_displayed:
+            console_write( string )
+        else:
+            sublime.error_message(u'Package Control\n\n%s\n%s' % (
+                string,
+                u'''
+                    If there will be new error messages on the next seconds,
+                    they will be show on the Sublime Text console
+                '''
+                ) )
+
+            is_error_recentely_displayed = True
+
+            # Enable the message dialog after x.x seconds
+            thread = Timer(60.0, _restart_error_messages)
+            thread.start()
```

Comment:
I am wondering if this can be handled using `sublime.set_timeout()` here?

Reply:
Sure, but I do not what is the difference. Usually I prefer to avoid sublime API, because with my experience it seems more bugged than the Python.